packages/button/src/element.ts

```diff
@@ -69,6 +82,12 @@ export class ButtonElement extends ButtonBase implements ButtonElementProps {
   protected renderIcon(): TemplateResult {
     return html`<fwc-icon class="mdc-button__icon" .icon=${this.icon}></fwc-icon>`;
   }
+
+  protected _onClick() {
+    if (this.links) {
+      this.links[0].click();
+    }
```

Comment:
we need scheck:
- [ ] check if the selector is actually an array
- [ ] check if the array contains a HTMLLinkElement